charts/sysdig-deploy/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.0.7
+
+### Minor changes
```

Comment:
you got quite some change here. I'd say it's "Major changes"

Reply:
I agree, I have changed it